webapp/src/main/java/com/google/sps/servlets/UploadServlet.java

```diff
@@ -37,10 +47,14 @@
   private static final int MAX_FILE_SIZE = 1024 * 1024 * 40; // 40MB
   private static final int MAX_REQUEST_SIZE = 1024 * 1024 * 50; // 50MB
 
+  // Log any issues
+  private static final Logger LOGGER = Logger.getLogger(UploadServlet.class.getName());
```

Comment:
s/LOGGER/logger/

Reply:
Is this a link to a google resource? Because I do not have access to it (403 error)

Reply:
Ok will rename LOGGER to logger